tests/resources/transaction/test_pushtx.py

```diff
@@ -56,7 +56,14 @@ def test_get(self):
         tx.parents = self.manager.get_new_tx_parents(tx.timestamp)
         tx.resolve()
 
-        response = yield self.web.get('push_tx', {b'hex_tx': bytes(tx.get_struct().hex(), 'utf-8')})
+        push_tx_fn = self.web.post if is_post else self.web.get
+        hex_param = 'hex_tx' if is_post else b'hex_tx'
+        force_param = 'force' if is_post else b'force'
+
+        tx_hex = tx.get_struct().hex()
+        hex_data = tx_hex if is_post else bytes(tx_hex, 'utf-8')
```

Comment:
Why do we have different things? Are we sending binaries in the HTTP POST's body?

Reply:
No, the GET requires bytes and the POST doesn't